subsys/bluetooth/shell/gatt.c

```diff
@@ -236,6 +236,7 @@ int cmd_gatt_read(int argc, char *argv[])
 	read_params.handle_count = 1;
 	read_params.single.handle = strtoul(argv[1], NULL, 16);
 
+	read_params.single.offset = 0;
```

Comment:
Minor thing about vertical whitespace: please keep the grouping of read_params assignments, i.e. no empty line above the "offset" one and empty line before the if statement below.